### PR TITLE
Added Lisburn and Castlereagh City Council

### PR DESCRIPTION
**Pull request by benlyall**
Addition of a new source for Lisburn and Castlereagh City Council.

`README.md` and `info.md` have both been updated by running `update_docu_links.py`.

The new module and it's corresponding doc `.md` have been provided and when running the tests, the following is the output:
```
❯ ./test_sources.py -s lisburn_castlereagh_gov_uk -i -l
Testing source lisburn_castlereagh_gov_uk ...
  found 6 entries for Test_001
    2023-04-26 : Refuse [mdi:trash-can]
    2023-05-03 : Recycling [mdi:recycle]
    2023-05-03 : Garden [mdi:leaf]
    2023-05-10 : Refuse [mdi:trash-can]
    2023-05-17 : Recycling [mdi:recycle]
    2023-05-17 : Garden [mdi:leaf]
  found 4 entries for Test_002
    2023-05-01 : Refuse [mdi:trash-can]
    2023-05-08 : Recycling [mdi:recycle]
    2023-05-08 : Garden [mdi:leaf]
    2023-05-15 : Refuse [mdi:trash-can]
  found 4 entries for Test_003
    2023-05-01 : Refuse [mdi:trash-can]
    2023-05-08 : Recycling [mdi:recycle]
    2023-05-08 : Garden [mdi:leaf]
    2023-05-15 : Refuse [mdi:trash-can]
  found 4 entries for Test_004
    2023-04-28 : Refuse [mdi:trash-can]
    2023-05-05 : Recycling [mdi:recycle]
    2023-05-05 : Garden [mdi:leaf]
    2023-05-12 : Refuse [mdi:trash-can]
```

### CODE REVIEW

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/lisburn_castlereagh_gov_uk.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+from datetime import datetime
+
+from waste_collection_schedule import Collection
+
+TITLE = "Lisburn and Castlereagh City Council" # Title will show up in README.md and info.md
+DESCRIPTION = "Source for the Lisburn and Castlereagh City Council"  # Describe your source
+URL = "https://lisburncastlereagh.gov.uk/"  # Insert url to service homepage. URL will show up in README.md and info.md
+TEST_CASES = {
+    "Test_001": {"postcode": "BT28 1AG", "house_number": "19A"},
+    "Test_002": {"postcode": "BT26 6AB", "house_number": "31"},
```
Comment (5ila5) [inline custom_components/waste_collection_schedule/waste_collection_schedule/source/lisburn_castlereagh_gov_uk.py:14]: Your Source should also work with integer `house_number` the user should not care about how to format the number. 
Please add a Test for an integer house_number.

Comment (benlyall) [reply custom_components/waste_collection_schedule/waste_collection_schedule/source/lisburn_castlereagh_gov_uk.py:14]: Good point. Have updated handling of the house number and added a test.

Comment (benlyall) [reply custom_components/waste_collection_schedule/waste_collection_schedule/source/lisburn_castlereagh_gov_uk.py:14]: New testing output:
```❯ ./test_sources.py -s lisburn_castlereagh_gov_uk -i -l                        
Testing source lisburn_castlereagh_gov_uk ...
  found 5 entries for Test_001
    2023-05-03 : Recycling [mdi:recycle]
    2023-05-03 : Garden [mdi:leaf]
    2023-05-10 : Refuse [mdi:trash-can]
    2023-05-17 : Recycling [mdi:recycle]
    2023-05-17 : Garden [mdi:leaf]
  found 4 entries for Test_002
    2023-05-01 : Refuse [mdi:trash-can]
    2023-05-08 : Recycling [mdi:recycle]
    2023-05-08 : Garden [mdi:leaf]
    2023-05-15 : Refuse [mdi:trash-can]
  found 4 entries for Test_003
    2023-05-01 : Refuse [mdi:trash-can]
    2023-05-08 : Recycling [mdi:recycle]
    2023-05-08 : Garden [mdi:leaf]
    2023-05-15 : Refuse [mdi:trash-can]
  found 4 entries for Test_004
    2023-05-01 : Refuse [mdi:trash-can]
    2023-05-08 : Recycling [mdi:recycle]
    2023-05-08 : Garden [mdi:leaf]
    2023-05-15 : Refuse [mdi:trash-can]
  found 5 entries for Test_005
    2023-05-05 : Recycling [mdi:recycle]
    2023-05-05 : Garden [mdi:leaf]
    2023-05-12 : Refuse [mdi:trash-can]
    2023-05-19 : Recycling [mdi:recycle]
    2023-05-19 : Garden [mdi:leaf]```

**Comment by 5ila5** [comment]
Reformatted the python file and Fixed one Typo (l 89, was 85 before reformatting)
Thanks for your contribution.

**Pull request merged**